patroni/__main__.py

```diff
@@ -259,6 +259,8 @@ def process_arguments() -> Namespace:
                        help='Generate a Patroni yaml configuration file for a running instance')
     parser.add_argument('--dsn', help='Optional DSN string of the instance to be used as a source \
                                     for config generation. Superuser connection is required.')
+    parser.add_argument('--ignore-listen-port', '-i', action='store_true',
+                        help='Ignore if the ports in `listen` are already bound by some process.')
```

Comment:
Please update the help message to make it clear that this option works only for config validation.

Reply:
Besides that, please update the `docstring` of this function.